includes/admin/payments/view-order-details.php

```diff
@@ -105,7 +105,7 @@
 										<div class="edd-admin-box-inside">
 											<p>
 												<span class="label"><?php _e( 'Date:', 'easy-digital-downloads' ); ?></span>&nbsp;
-												<input type="text" name="edd-payment-date" value="<?php echo esc_attr( date( 'm/d/Y', $payment_date ) ); ?>" class="medium-text edd_datepicker"/>
+												<input type="text" name="edd-payment-date" value="<?php echo esc_attr( date( 'Y-m-d', $payment_date ) ); ?>" class="medium-text edd_datepicker" placeholder="<?php echo edd_get_date_picker_format(); ?>" />
```

Comment:
Why the change in entered format? The datepicker reformats to Y-m-d on `$_POST` anyway, doesn't it?

Reply:
This is so that the date matches the placeholder text and the returned format of jQuery UI Datepicker.